cloud-sql/mysql/sqlalchemy/deployment.yaml

```
@@ -38,10 +38,18 @@ spec:
         env:
         - name: PORT
           value: "8080"
+        # This project uses environment variables to determine
+        # how you would like to run your application
+        # To use the Python connector (recommended) - use INSTANCE_CONNECTION_NAME (proj:region:instance)
+        # To use TCP - Setting INSTANCE_HOST will use TCP (e.g., 127.0.0.1)
+        # To use Unix, use INSTANCE_UNIX_SOCKET (e.g., /cloudsql/proj:region:instance)
         - name: INSTANCE_HOST
           value: "127.0.0.1"
         - name: DB_PORT
-          value: "3306"  
+          value: "3306"
+        # To use Automatic IAM Authentication (recommended)
+        # use DB_IAM_USER instead of DB_USER

```

I think this makes it a bit more straightforward on the actual option being recommended but a very minor recommendation :P
```suggestion
        # To use Automatic IAM Authentication 
        # use DB_IAM_USER instead of DB_USER (recommended)
```